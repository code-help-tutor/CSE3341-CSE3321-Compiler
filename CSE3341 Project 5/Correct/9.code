procedure t9 and
	procedure A(a, b) is
		record n;
	begin
		a[0] := a[0]+b[0];
		b := record n;
	end
	procedure S(a, b) is
		record n;
	begin
		a[0] := a[0]-b[0];
		begin A(a, b);
		b := record n;
	end
is
	record x;
	record y;
begin
	x := new record[1];
	x[0] := 1;
	y := new record[1];
	y := 1+x;
	begin S(x, y);
	out(x);
end
