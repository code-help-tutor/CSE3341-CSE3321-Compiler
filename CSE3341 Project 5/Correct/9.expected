gc:1
gc:2
1
gc:1
gc:0
